liquibase-service/src/main/resources/db/rdb_modern/routines/023-sp_f_std_page_case_postprocessing-001.sql

```diff
@@ -0,0 +1,685 @@
+AS
+BEGIN
+BEGIN TRY
+
+   declare @rowcount_no bigint;
+   declare @proc_step_no float = 0;
+   declare @proc_step_name varchar(200) = '';
+   declare @batch_id bigint;
+   declare @create_dttm datetime2(7) = current_timestamp;
+   declare @update_dttm datetime2(7) = current_timestamp;
+   declare @dataflow_name varchar(200) = 'F_STD_PAGE_CASE';
+   declare @package_name varchar(200) = 'F_STD_PAGE_CASE';
+
+   SET @batch_id = cast((format(getdate(),'yyMMddHHmmss')) as bigint);
+
+    INSERT INTO [dbo].[job_flow_log]
+    (batch_id
+        ,[create_dttm]
+        ,[update_dttm]
+        ,[Dataflow_Name]
+        ,[package_Name]
+        ,[Status_Type]
+        ,[step_number]
+        ,[step_name]
+        ,[msg_description1]
+        ,[row_count])
+    VALUES (@batch_id
+            ,@create_dttm
+            ,@update_dttm
+            ,@dataflow_name
+            ,@package_name
+            ,'START'
+            ,0
+            ,'SP_Start'
+            ,LEFT(@phc_id_list, 500)
+            ,0);
+
+
+    BEGIN TRANSACTION;
+            SET @proc_step_no = 1;
+            SET @proc_step_name = ' Generating PHC_UIDS_ALL';
+
+            IF OBJECT_ID('#PHC_CASE_UIDS_ALL', 'U') IS NOT NULL
+    drop table #PHC_CASE_UIDS_ALL
+    ;
+
+    SELECT
+        ni.public_health_case_uid  'PAGE_CASE_UID',
+            nicm.CASE_MANAGEMENT_UID,
+        ni.INVESTIGATION_FORM_CD,
+        ni.CD,
+        ni.LAST_CHG_TIME,
+        ni.ADD_TIME,
+        ni.investigator_id,
+        ni.physician_id,
+        ni.patient_id,
+        ni.person_as_reporter_uid,
+        ni.hospital_uid,
+        ni.ordering_facility_uid,
+        ni.ca_supervisorofphc_uid,
+        ni.closure_investgr_of_phc_uid,
+        ni.dispo_fld_fupinvestgr_of_phc_uid,
+        ni.fld_fup_investgr_of_phc_uid,
+        ni.fld_fup_prov_of_phc_uid,
+        ni.fld_fup_supervisor_of_phc_uid,
+        ni.init_fld_fup_investgr_of_phc_uid,
+        ni.init_fup_investgr_of_phc_uid,
+        ni.init_interviewer_of_phc_uid,
+        ni.interviewer_of_phc_uid,
+        ni.surv_investgr_of_phc_uid,
+        ni.fld_fup_facility_of_phc_uid,
+        ni.org_as_hospital_of_delivery_uid,
+        ni.per_as_provider_of_delivery_uid,
+        ni.per_as_provider_of_obgyn_uid,
+        ni.per_as_provider_of_pediatrics_uid,
+        ni.org_as_reporter_uid
+    INTO
+        #PHC_CASE_UIDS_ALL
+    FROM
+        rdb_modern.dbo.nrt_investigation  ni
+            LEFT OUTER JOIN rdb_modern.dbo.nrt_investigation_case_management nicm ON	ni.public_health_case_uid = nicm.public_health_case_uid
+            LEFT OUTER JOIN NBS_SRTE.dbo.CONDITION_CODE cc ON 	cc.CONDITION_CD= ni.CD AND	cc.INVESTIGATION_FORM_CD
+            NOT IN 	( 'bo.','INV_FORM_BMDGBS','INV_FORM_BMDGEN','INV_FORM_BMDNM','INV_FORM_BMDSP','INV_FORM_GEN','INV_FORM_HEPA','INV_FORM_HEPBV','INV_FORM_HEPCV','INV_FORM_HEPGEN','INV_FORM_MEA','INV_FORM_PER','INV_FORM_RUB','INV_FORM_RVCT','INV_FORM_VAR')
```

Comment:
In the post-processing sprocs we aren't including the database name when referencing tables that live in `rdb_mdoern` ,  just the schema name and table name. The SRTE reference is fine of course.

Reply:
yes that is correct, i will have to get rid of these